test/gtest/common/test_helpers.h

```diff
@@ -743,11 +743,11 @@ class auto_ptr {
         _t h; \
         ucs_status_t status = _ctor(__VA_ARGS__, &h); \
         if (status == UCS_ERR_UNSUPPORTED) { \
-            UCS_TEST_SKIP_R(std::string("Unsupported operation: ") + \
-                            UCS_PP_MAKE_STRING(_ctor)); \
+            _handle.reset(); \
```

Comment:
why needed? can we avoid using unsupported flow? check some cap flag?

Reply:
will check